src/components/Button.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function Button(props) {
+  const { value, isOperator, isZero } = props;
+  if (isOperator) {
+    return (
+      <div className="op-btn">
+        {value}
+      </div>
+    );
+  }
+  return (
+    <div className={isZero ? 'btn zero' : 'btn'}>
+      {value}
+    </div>
+  );
+}
+
```

Comment:
- [ ] Button component should accept `a button name` as an incoming prop ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/button/acceptProp.png))